service/frontend/workflow_handler.go

```diff
@@ -3363,14 +3304,12 @@ func (wh *WorkflowHandler) DeleteSchedule(ctx context.Context, request *workflow
 		return nil, err
 	}
 
-	_, err = wh.historyClient.TerminateWorkflowExecution(ctx, &historyservice.TerminateWorkflowExecutionRequest{
-		NamespaceId: namespaceID.String(),
-		TerminateRequest: &workflowservice.TerminateWorkflowExecutionRequest{
-			Namespace:         request.Namespace,
-			WorkflowExecution: &commonpb.WorkflowExecution{WorkflowId: workflowID},
-			Reason:            "terminated by DeleteSchedule",
-			Identity:          request.Identity,
-		},
+	execution := &commonpb.WorkflowExecution{WorkflowId: workflowID}
+	_, err = wh.historyClient.DeleteWorkflowExecution(ctx, &historyservice.DeleteWorkflowExecutionRequest{
```

Comment:
what if we had this sequence of events:
1. workflow was started and completed (idle)
2. workflow was re-started, but call to DeleteWorkflow of previous failed
3. call to DeleteSchedule. it'll delete the latest one, but then it'll "expose" the one before that

probably this needs to delete all executions, not just the latest? (though of course there will usually be only one)